src/services/navigator/helpers.test.ts

```diff
@@ -2,7 +2,7 @@ import * as DomErrors from '../dom/errors';
 import * as Errors from './errors';
 import * as Namespaces from '../namespaces';
 import * as Types from './types';
-import * as TypesLegacy from '../../types-legacy';
+import * as TypesLegacy from '../../types';
```

Comment:
is this change correct? It doesn't look like this PR renames the file.

Reply:
yes, due to this tests were failing as this file doesn't exist